Formula/seqan@3.rb

```diff
@@ -15,15 +15,25 @@ class SeqanAT3 < Formula
   end
 
   depends_on "cmake" => :build
-  depends_on "doxygen" => :build
   depends_on "xz" => :build
+  depends_on "gcc@8"
```

Comment:
```suggestion
  depends_on "gcc@10"
```
We usually use the most modern available GCC, when the default GCC is too old.